src/Conv.sol

```diff
@@ -58,17 +59,17 @@ contract Conv {
     /// @notice Fetches the yearly bps rate for a given per second rate
     /// @param ray The per second rate to get the rate for
     /// @return bps The annual rate value
-    function nrut(uint256 ray) public pure returns (uint256 bps) {
+    function back(uint256 ray) public pure returns (uint256 bps) {
         // Convert per-second rate to per-year rate using rpow
         uint256 yearlyRate = _rpow(ray, 365 days);
         // Subtract RAY to get the yearly rate delta and convert to basis points
         // Add RAY/2 for rounding: ensures values are rounded up when >= 0.5 and down when < 0.5
         return ((yearlyRate - RAY) * 10000 + RAY / 2) / RAY;
     }
 
-    /// @notice Exponentiate `x` to `n` by squaring
-    /// @param x The base
-    /// @param n The exponent
+    /// @notice Exponentiate `x` (RAY, 27 decimal places) to `n` () by squaring
```

Comment:
It's a bit redundant now. It's fine to leave the explanation only in the parameters below.
```suggestion
    /// @notice Exponentiate `x` to `n` by squaring
```